libvcx/src/api_lib/api_handle/credential_def.rs

```diff
@@ -249,41 +285,19 @@ pub mod tests {
         let (schema_id, _) = create_and_write_test_schema(utils::constants::DEFAULT_SCHEMA_ATTRS);
         let did = settings::get_config_value(settings::CONFIG_INSTITUTION_DID).unwrap();
 
-        let revocation_details = json!({"support_revocation": true, "tails_file": get_temp_dir_path("tails.txt").to_str().unwrap(), "max_creds": 2, "tails_url": utils::constants::TEST_TAILS_URL.to_string()}).to_string();
+        let revocation_details = json!({"support_revocation": true, "tails_file": get_temp_dir_path("tails.txt").to_str().unwrap(), "max_creds": 2}).to_string();
         let handle = create_and_publish_credentialdef("1".to_string(),
-                                                      "test_tails_url_written_to_ledger".to_string(),
                                                       did,
                                                       schema_id,
                                                       "tag1".to_string(),
-                                                      revocation_details).unwrap();
+                                                      revocation_details,
+                                                      Some(utils::constants::TEST_TAILS_URL.to_string())).unwrap();
         let rev_reg_def = get_rev_reg_def(handle).unwrap().unwrap();
         let rev_reg_def: serde_json::Value = serde_json::from_str(&rev_reg_def).unwrap();
         let _rev_reg_id = get_rev_reg_id(handle).unwrap();
         assert_eq!(rev_reg_def["value"]["tailsLocation"], utils::constants::TEST_TAILS_URL.to_string());
     }
 
-    #[cfg(feature = "pool_tests")]
-    #[test]
-    fn test_tails_base_url_written_to_ledger() {
```

Comment:
Perhaps should not be deleted but adjusted to use our new created/publish methods?

Reply:
This particular test is irrelevant now but I agree we should add at least one new test

Reply:
I think the test above the scenario appropriately?

Reply:
Oh yeah, that looks good!